projector-awt/gradle.properties

```diff
@@ -0,0 +1 @@
+version=1.0-SNAPSHOT
```

Comment:
Can we use the same version for the projector-awt as for projector-server for now?